src/exception/call_stack.cr

```diff
@@ -22,9 +22,12 @@ struct Exception::CallStack
   # Compute current directory at the beginning so filenames
   # are always shown relative to the *starting* working directory.
   CURRENT_DIR = begin
-    dir = Process::INITIAL_PWD
-    dir += File::SEPARATOR unless dir.ends_with?(File::SEPARATOR)
-    dir
+    if dir = Process::INITIAL_PWD
+      dir += File::SEPARATOR unless dir.ends_with?(File::SEPARATOR)
+      dir
+    else
+      ""
+    end
```

Comment:
Empty string here seems ambiguous, why not return `nil` in the same vein as `Process::INITIAL_PWD` does it?

Reply:
The only place it's being used is https://github.com/crystal-lang/crystal/blob/f991977999e61fc18922fc00c3582552d9bf1ec9/src/exception/call_stack.cr#L177-L178